src/app/tic-tac-toe.component.ts

```diff
@@ -141,7 +141,7 @@ export class TicTacToeComponent {
     }
 
     // Diagonals: top-left to bottom-right
-    for (let rowStart = 0; rowStart < this._size - 4; rowStart++) {
+    for (let rowStart = 0; rowStart < this._size - 4; rowStart++) { // review
```

Comment:
This will not work if the `winCondition` changes -- for `winCondition > 5` it will stop being optimized and for `winCondition < 5` it will not work in (in this case top-right and bottom-left) corners.